packages/SwingSet/test/vat-admin/slow-termination/bootstrap-slow-terminate.js

```diff
@@ -21,11 +22,22 @@ export function buildRootObject(_vatPowers) {
       }
 
       // set up 20 "dude exports, bootstrap imports" c-list entries
-
       for (let i = 0; i < 20; i += 1) {
         myImports.push(await E(root).sendExport());
       }
 
+      // also 10 imported promises
+      for (let i = 0; i < 10; i += 1) {
+        await E(root).acceptImports(makePromiseKit().promise);
```

Comment:
To be clear, these eventual sends also result in imported / externally decided promises but since they're immediately settled, they do not participate in any cleanup?

Reply:
Right, the `acceptImports` result promises are settled (and the notifies delivered) during the bootstrap run, driven by the `await controller.run()` on line 110 of `slow-termination.test.js`. The `makePromiseKit().promise` *payload* of those messages are never resolved (they remain decided by vat-bootstrap, but we've thrown away the settlement functions), so they'll still be in the c-list when the vat is terminated. The exports will be rejected when the target vat is terminated, and removed from the c-list by the `run()` on line 191. But the imports will still be sitting there when the slow deletion happens.